videoframes.py

```diff
@@ -349,3 +368,15 @@ def resizeEvent(self, event):
             rect.width() / 2 - backwardButton_rect.width() / 2 - 0.5 * backwardButton_rect.size().width(),
             rect.height() - backwardButton_rect.size().height()
         )
+
+    def enterEvent(self, event):
+        self.pause_button.show()
```

Comment:
We have a bit of code duplication. Perhaps `pause_button` and `volume_slider` can be put inside of `_VideoFrame`?